dll/shellext/mydocs/mydocs.cpp

```diff
@@ -16,6 +16,43 @@ END_OBJECT_MAP()
 CComModule gModule;
 LONG g_ModuleRefCnt = 0;
 
+static BOOL
+CreateEmptyFile(LPCWSTR pszFile)
+{
+    HANDLE hFile;
+    hFile = CreateFileW(pszFile, GENERIC_WRITE, FILE_SHARE_READ, NULL,
+                        CREATE_ALWAYS, FILE_ATTRIBUTE_NORMAL, NULL);
+    CloseHandle(hFile);
```

Comment:
if `hFile == INVALID_HANDLE_VALUE` this might cause an exception (based on debug settings)

Reply:
A similar bug is present there too:
https://github.com/reactos/reactos/commit/5538facfdd0edd11038f9dd00bb4a6afec440403#diff-7269d101ea680a5d7880e88eb7a1680cR2224-R2226

Reply:
OK.